ssf-words.txt

```diff
@@ -78,7 +78,7 @@ taskrun
 Tekton
 tekton
 tektoncd
-thesecuresoftwarefactory
+buildsec
```

Comment:
Does this list need to be sorted?

Reply:
No, it just makes it nicer to read/update.